docs/bcfusion/validators/creation.md

```diff
@@ -1,2 +1,137 @@
-This page will guide validators to create new native BSC validators.
-Please be patient for updates.
\ No newline at end of file
+import Validator1 from '../../assets/bcfusion/create-validator1.png';
+import Validator2 from '../../assets/bcfusion/create-validator2.png';
+import Validator3 from '../../assets/bcfusion/create-validator3.png';
+import Validator4 from '../../assets/bcfusion/create-validator4.png';
+
+# Validator Creation
+
+If you are migrating old validators created on
+the BNB Beacon Chain, please refer to [this document](migrations.md) for more information.
```

Comment:
I think `migrations.md` should be only suggested when the dapp is not available or the validator like to use cli.

I think `migrating old validators` can still use this dapp, just create the validator with proper `identity` and then migrate the delegation.

Reply:
Sure. Updated the document.